### PR TITLE
Add information about pending signatures

### LINKED ISSUE #355: Task: add information about pending signatures during a DAS Bootstrap ceremony
### What is the task about?

Add information to the user about pending signatures during a DAS Bootstrap ceremony.
I imagine this will be at the end of the bootstrap ceremony so that the user understands that:
1. the root metadata is not yet considered trustworthy
2. X amount of signatures are needed to finish the bootstrap process

This is important as there are multiple questions and steps in the bootstrap ceremony and it's possible to forget what was the threshold and how many keys were actually loaded and the user can be left with the impression he has finished with the bootstrap ceremony.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ivanayov**
This change outputs a warning and a number of pending root signatures during DAS bootstrap ceremony

Fixes #355

# Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

# Additional requirements
- [x] Tests have been added for the bug fix or new feature
- [n/a ] Docs have been added for the bug fix or new feature

# Code of Conduct
By submitting this PR, you agree to follow our [Code of Conduct](CODE_OF_CONDUCT.rst).

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: repository_service_tuf/cli/admin/ceremony.py [hunk 1]
```diff
@@ -628,6 +630,15 @@ def _run_ceremony_steps(save: bool) -> Dict[str, Any]:
...
     metadata = tuf_management.initialize_metadata()
 
+    # Inform user for pending signatures
+    vals = metadata.values()
+    root: Metadata[Root] = vals[Roles.ROOT]
+    if root.signatures < setup.threshold[Roles.ROOT]:
+            console.print(
+                "Root is not trustworhy yet. %d pending signatures are left",
+                setup.threshold[Roles.ROOT] - len(root.signatures)
+            )
```
Comment (kairoaraujo) [inline repository_service_tuf/cli/admin/ceremony.py:640]: It's preferable to use [f-strings](https://peps.python.org/pep-0498/) and you can reuse the calculation you did above
```suggestion
        console.print(
            f"Root is not trustworthy yet. {pending_signatures} pending signatures are left."
        )
```

**Review by kairoaraujo — CHANGES_REQUESTED** [review]
Hi @ivanayov,

Thank you for your first contribution to RSTUF :tada:  It is amazing having you here!

I left a few comments here, feel free to discuss them.

Tips:
You can run `make reformat` to fix PEP8 and formatting.

We also recommend [enable the `pre-commit`](https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/main/CONTRIBUTING.rst#running-checks-with-pre-commit)

File: repository_service_tuf/cli/admin/ceremony.py [hunk 2]
```diff
@@ -628,6 +630,15 @@ def _run_ceremony_steps(save: bool) -> Dict[str, Any]:
     tuf_management = TUFManagement(setup, save)
     metadata = tuf_management.initialize_metadata()
 
+    # Inform user for pending signatures
+    vals = metadata.values()
+    root: Metadata[Root] = vals[Roles.ROOT]
+    if root.signatures < setup.threshold[Roles.ROOT]:
```
[the remark this answers is not in the document]
Comment (MVrachev) [reply repository_service_tuf/cli/admin/ceremony.py:636]: This also has a benefit it makes it clearer about what the math: `setup.threshold[Roles.ROOT] - len(metadata[Roles.ROOT].signatures)` calculates.

File: tests/unit/cli/admin/test_ceremony.py [hunk 3]
```diff
@@ -371,6 +372,41 @@ def test_ceremony_validation_reconfigure_root_keys(
...
+        ceremony.setup = test_setup
+        input_step1, input_step2, input_step3, input_step4 = test_inputs
+        # Setting threshold to a high value to guarantee there are stil
+        # signatures needed to finish the bootstrap process
+        input_step1 = [
+            "y",  # Do you want more information about roles and responsibilities?  # noqa
+            "y",  # Do you want to start the ceremony?
+            "",  # What is the metadata expiration for the root role?(Days)
+            "",  # What is the number of keys for the root role? (2)
+            "20",  # What is the key threshold for root role signing?
```
[the remark this answers is not in the document]
Comment (ivanayov) [reply tests/unit/cli/admin/test_ceremony.py:388]: It's a good catch! We could expand the issue or I can do in a follow-up?

[the remark this answers is not in the document]
Comment (kairoaraujo) [reply tests/unit/cli/admin/test_ceremony.py:388]: As you wish 😄

**Comment by ivanayov** [comment]
Pushed again, as previously I pushed without the changes by mistake.

**Comment by ivanayov** [comment]
@kairoaraujo It has the sslib fix now and is ready to run

File: repository_service_tuf/cli/admin/ceremony.py [hunk 4]
```diff
@@ -628,6 +628,16 @@ def _run_ceremony_steps(save: bool) -> Dict[str, Any]:
...
     metadata = tuf_management.initialize_metadata()
 
+    # Inform user for pending signatures
+    pending_signatures: int = setup.threshold[Roles.ROOT] - len(
+        metadata[Roles.ROOT.value].signatures
+    )
+    if pending_signatures > 0:
+        console.print(
+            "Root is not trustworthy yet."
+            f" {pending_signatures} pending signatures are left.",
```
Comment (MVrachev) [inline repository_service_tuf/cli/admin/ceremony.py:638]:
```suggestion
            "Root is not trustworthy yet,"
            f" {pending_signatures} pending signature(s) left.",
```

**Comment by ivanayov** [comment]
Forgot to update the test after changing the layout.

**Comment by ivanayov** [comment]
Forse-pushed the test update.

**Review by MVrachev — APPROVED** [review]

**Comment by MVrachev** [comment]
@kairoaraujo you are required to give approval otherwise we cannot merge.

**Review by kairoaraujo — APPROVED** [review]

**Pull request merged**